### PR TITLE
-steal- import config from puppetlabs

### PR DESCRIPTION
**Pull request by igalic**

### CODE REVIEW

File: moduleroot/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+notifications:
+  email: false
+language: ruby
+bundler_args: --without development system_tests
+sudo: false
+before_install: rm Gemfile.lock || true
+script: bundle exec rake test
```
Comment (cmurphy) [inline moduleroot/.travis.yml]: Not `rake spec`?

Comment (cmurphy) [inline moduleroot/.travis.yml]: nvm I see the rakefile

**Comment by cmurphy** [comment]
:+1:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+ModuleSync Configs
+==================
+
+Module sync configurations for Puppet-Community Modules
+
+This is the repository where you wanna commit all the ***fun*** changes when we
+need to add new boilerplate code.
```
Comment (nibalizer) [inline README.md]: Please add instructions on how to use this thing.

File: config_defaults.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+.travis.yml:
+  script: "\"bundle exec rake validate lint spec SPEC_OPTS='--format documentation'\""
```
Comment (nibalizer) [inline config_defaults.yml]: What are the slashes for?

File: moduleroot/spec/acceptance/nodesets/centos-64-x64-pe.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+HOSTS:
+  centos-64-x64:
+    roles:
+      - master
```
Comment (nibalizer) [inline moduleroot/spec/acceptance/nodesets/centos-64-x64-pe.yml]: these roles confuse me

File: moduleroot/spec/acceptance/nodesets/centos-59-x64.yml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+HOSTS:
+  centos-59-x64:
+    roles:
+      - master
```
Comment (nibalizer) [inline moduleroot/spec/acceptance/nodesets/centos-59-x64.yml]: master really?

File: README.md (hunk 2 above)
Comment (igalic) [inline README.md]: @cmurphy how do we use this (assuming if it's rewritten it stays kinda the same ;)

File: config_defaults.yml (hunk 3 above)
Comment (igalic) [inline config_defaults.yml]: they are copy pasted from @cmurphy. for one, for the other, i seem to have forgotten to add a template line moduleroot/.travis.yml

File: moduleroot/spec/acceptance/nodesets/centos-64-x64-pe.yml (hunk 4 above)
Comment (igalic) [inline moduleroot/spec/acceptance/nodesets/centos-64-x64-pe.yml]: this, again, is copy/pasta from @puppetlabs.

File: moduleroot/spec/acceptance/nodesets/centos-59-x64.yml (hunk 5 above)
Comment (igalic) [inline moduleroot/spec/acceptance/nodesets/centos-59-x64.yml]: let's just get rid of 5.9

Comment (nibalizer) [inline moduleroot/spec/acceptance/nodesets/centos-59-x64.yml]: Hooray!

**Pull request merged**